### PR TITLE
REVIEW: NEXUS-5615 update UI to indicate first/last name are required and add a last name t...

### PR DESCRIPTION
**Pull request by kellyrob99**
...o default admin user for new deployments

https://issues.sonatype.org/browse/NEXUS-5615

### CODE REVIEW

**Comment by jdillon** [comment]
Please cross link github and jira... so its easier to move between them.

https://issues.sonatype.org/browse/NEXUS-5615

File: nexus-core/src/main/resources/META-INF/security/security.xml [hunk 1]
```diff
@@ -21,6 +21,7 @@
 		<user>
 			<id>admin</id>
 			<firstName>Administrator</firstName>
+            <lastName>Nexus</lastName>
```
Comment (jdillon) [inline nexus-core/src/main/resources/META-INF/security/security.xml]: probably should be:

```
<lastName>User</lastName>
```

... which matches the others closer.

indent is also off but so is this entire file :-\

**Comment by jdillon** [comment]
This doesn't actually fix existing installations... but not sure we care too much about those, since hopefully they have deleted this user ;-)

**Comment by kellyrob99** [comment]
I considered updating existing installations, but since there's no functionality that is broken as-s it seemed overkill for such a small issue.
If anyone feels strongly about that, I'd welcome the opportunity to start learning about how to upgrade the configuration xml files properly.

**Comment by jdillon** [comment]
@kellyrob99 its not worth the trouble don't bother.

**Comment by kellyrob99** [comment]
+1 by CI https://bamboo.zion.sonatype.com/browse/NXO-OSSF23-1

**Comment by adreghiciu** [comment]
+1

**Comment by jdillon** [comment]
+1

**Pull request merged**